DESCRIPTION

```diff
@@ -33,6 +33,7 @@ Imports:
     shinyjs,
     stats,
     teal.logger (>= 0.1.1),
+    teal.code (>= 0.4.1.9003),
```

Comment:
can you please also add it below in `Config/Needs/verdepcheck` for verdepcheck tests